src/crimson/mon/MonClient.cc

```diff
@@ -193,7 +195,9 @@ Connection::authenticate(epoch_t epoch,
     auth = create_auth(m, name, want_keys);
     global_id = m->global_id;
     switch (auto p = m->result_bl.cbegin();
-            auth->handle_response(m->result, p)) {
+            auth->handle_response(m->result, p,
+#warning fix crimson: session_key, connection_secret
+				  nullptr, nullptr)) {
```

Comment:
Need to fix the warning

Reply:
these are breadcrumbs for @tchaikov et al to fix up later

Reply:
I get the idea, but...well, we have not been successful at that in the past, and instead have just polluted build output with warnings that need to be ignored for months or years at a time. It looks like we get about 70 warnings out of the "fixme need to set module_type for seastar" in the build log. Maybe we can use a different breadcrumb?